plenum/test/bls/test_bls_bft_replica.py

```diff
@@ -295,6 +315,19 @@ def test_validate_commit_incorrect_sig(bls_bft_replicas, pre_prepare_with_bls):
             assert status == BlsBftReplica.CM_BLS_SIG_WRONG
 
 
+def test_validate_commit_incorrect_signed_without_pool_state_root(bls_bft_replicas, multi_signature):
```

Comment:
Why the signature is incorrect in this case?

Reply:
Signature is correct in this case. Fixed wrong method name to `test_validate_commit_signature_without_pool_state_root`